CHANGELOG.md

```diff
@@ -1,12 +1,20 @@
 # Changelog
 
+## 0.31.2
+
+### Patch Changes
+
+- ef6e936: - **feat**: added connect wallet functionality to Swap component for disconnected users. By @abcrane123 #1173
```

Comment:
is this supposed to have the commit? @abcrane123

Reply:
no you can delete that

Reply:
we can clean that up in next PR